### ISSUE TITLE
Add commandline flag to not delete the prefixlist first

### ISSUE THREAD

**Issue by crami** [bgp/bgpq4#60]
I call bgpq4 twice once for normal filtering and one with -r 32 -R 32 for black holing, I use the same object name. It would be nice if on the second call i could suppress the "no ...."

Example:
bgpq4 -S $sources -A -4 -R {maxlen4} -l {name4} {object}
bgpq4 -S $sources -4 -r 32 -R 32 -l {name4} {object}

**Comment by job** [comment]
what about using fgrep?

```
bgpq4 .... -l {name4} | fgrep -v "no ip prefix-list {name4}"
```

**Comment by jkroonza** [comment]
I agree. Especially when using this as a feed into other mechanisms, eg:

```
rtr(config)#ipv6 prefix-list foo-v6 source https://routes.uls.co.za/iplist/as=327767/fmt=arista/ipv=6/max=48/permit=48/
```

Where basically that just remaps into a bgpq4 command at the back ... in this specific case we only want the seq .. statements.  Filtered these already, but I do agree that some option to have bgpq4 not output these in the first place would be most helpful.
